pkg/asset/tls.go

```diff
@@ -60,21 +68,39 @@ func newCACert() (*rsa.PrivateKey, *x509.Certificate, error) {
 	return key, cert, err
 }
 
-func newAPIKeyAndCert(caCert *x509.Certificate, caKey *rsa.PrivateKey, apiCertIPs []string) (*rsa.PrivateKey, *x509.Certificate, error) {
+func newAPIKeyAndCert(caCert *x509.Certificate, caKey *rsa.PrivateKey, altNames tlsutil.AltNames) (*rsa.PrivateKey, *x509.Certificate, error) {
 	key, err := tlsutil.NewPrivateKey()
 	if err != nil {
 		return nil, nil, err
 	}
+	altNames.IPs = append(altNames.IPs, net.ParseIP("10.3.0.1"))
```

Comment:
this eliminates the requirement that 10.3.0.1 be passed as the last argument during rendering, correct? e.g. `--apiserver-cert-ip-addrs=172.15.0.21,10.3.0.1`